### PR TITLE
#21 Add download bee data as CSV

### PR DESCRIPTION
**Pull request by Detect**
Add dropdown under bee sensors to download as CSV

requires pytz on hive. run this:
`wget https://bootstrap.pypa.io/ez_setup.py -O - | python && easy_install --upgrade pytz && easy_install --upgrade six`

### CODE REVIEW

**Comment by lthiery** [comment]
It crashes when you ask for too many datapoints.

I queried 4-5 days of data that were gathered every minute: 4_24_60=5760 datapoints

The wheel spun for a little bit and then it went to "No data received" page. I tried this in Chrome.

**Comment by rjcorwin** [comment]
If it helps, node's core fs module has a method for appending strings to the end of a file to do that chunking.

```
fs.appendFile('beeId.csv', 'data to append', function (err) {

});

```

File: Beekeeper/lib/export_csv.py.lprof [hunk 1]
```diff
@@ -0,0 +1,3 @@
+�c_line_profiler
```
Comment (Detect) [inline Beekeeper/lib/export_csv.py.lprof]: delete file

File: Beekeeper/server.js [hunk 2]
```diff
@@ -9,13 +9,43 @@ var harvestHoneyJars = require('./lib/HarvestHoneyJars.js')
...
+
+  beeId = req.query.beeId;
+
+  if(!!beeId){
+    var python = require('child_process').spawn(
+    'python',
+    [Settings.path + "/Beekeeper/lib/export_csv.py", beeId]
+    );
+
+    res.setHeader('Content-disposition', 'attachment; filename=' + beeId + '.csv');
```
Comment (Detect) [inline Beekeeper/server.js]: add `bee_` to filename

**Pull request merged**